.zombienet-tests/utils.js

```diff
@@ -0,0 +1,20 @@
+const { ApiPromise } = require("@polkadot/api")
+
+async function connect(nodeName, networkInfo, isParachain) {
+  const { userDefinedTypes } = networkInfo.nodesByName[nodeName]
+  const customChainSpec = require(networkInfo.chainSpecPath)
+  const { createPolkadotJsScClient } = await import("@substrate/connect")
+  const scClient = createPolkadotJsScClient()
+  let provider
+  if (isParachain) {
+    await scClient.addChain(JSON.stringify(customChainSpec))
+    const customParachainSpec = require(networkInfo?.paras[100]?.spec)
```

Comment:
I guess that's not your decision, but shouldn't this field be named `chainSpecPath` instead of `spec`?

Reply:
Yes, I will change to `chainSpecPath` in zombienet.